zh/TOC.md

```diff
@@ -24,7 +24,9 @@
   - [配置备份](configure-backup.md)
   - [配置 Storage Class](configure-storage-class.md)
   - [配置 TiDB Binlog Drainer](configure-tidb-binlog-drainer.md)
-- [监控](monitor-a-tidb-cluster.md)
++ 监控
+  - [监控 TiDB 集群](monitor-a-tidb-cluster.md)
+  - [通过 TidbMonitor 监控 TiDB 集群](monitoring-by-tidb-monitor.md)
```

Comment:
Use `using` instead of `by` to keep the wording consistent with other file names. 
Please update the original file name.
```suggestion
  - [通过 TidbMonitor 监控 TiDB 集群](monitoring-using-tidb-monitor.md)
```